katas/content/multi_qubit_gates/preparing_bell_state/Solution.qs

```diff
@@ -1,4 +1,4 @@
-namespace Kata {
+namespace Kata.Solution {
```

Comment:
Are we placing reference implementation in Kata.Solution?
(We were discussing Kata.Reference, but Kata.Solution is just as good).

Reply:
I was thinking on `Kata.Verification` since now we won't have a standalone solution file.

Reply:
Updated to use `Kata.Verification.CheckSolution`.